src/services/queries/Users.js

```diff
@@ -1,9 +1,10 @@
 import gql from 'graphql-tag';
+import { userInfo } from '../fragments';
 
-export const getAllUsersQuery = gql`
+export const USERS_QUERY = gql`
```

Comment:
Is this query of use in the app? Like, you only need user by member_code/password maybe?

Reply:
No, it's not used on the APP, I have use it to test the integration with the GraphQL, but I should remove it hehe. Thanks for notifying me